src/actions/index.js

```diff
@@ -1,8 +1,17 @@
-// we'll need axios
+import axios from 'axios';
 
-// we'll need to create 3 different action types here.
-// one for fetching, one for success and one for failure
+export const FETCH_SWAPI_START = 'FETCH_SWAPI_START';
+export const FETCH_SWAPI_SUCCESS = 'FETCH_SWAPI_SUCCESS';
+export const FETCH_SWAPI_FAILURE = 'FETCH_SWAPI_FAILURE';
```

Comment:
perfect, I prefer this to `FETCH_START` and the like in case the app/functionality ever grows and there could be multiple things you fetch, fail, succeed, etc